catalog/core/catalog-core-api/src/main/java/ddf/catalog/Constants.java

```diff
@@ -125,6 +125,8 @@
 
   public static final String INGEST_LOGGER_NAME = "ingestLogger";
 
+  public static final String CDM_LOGGER_NAME = "cdmLogger";
```

Comment:
I am thinking this constant should be included somewhere in the CDM bundle instead of the catalog api since it is only used in that one bundle.